src/Pages/MyPage.tsx

```diff
@@ -22,7 +26,11 @@ export function MyPage(props:{stemNum:number, optionNum:number}) {
                     <MadeOption optionType='Distractor'/>
                 </MadeLists>
             </div>
-            <button>Log Out</button>
+            <button onClick={() => {
```

Comment:
I think it is better to define a logout function in parent component and pass the function to this component.

```tsx
const logout = useCallback(() => {
  googleLogout(); 
  setIsLoggedIn(false);
   navigate('/login');
}, [])
```

```tsx
<button onClick={props.logout}>Log Out</button>
```

Reply:
aha it is better to understand. 👍✅